ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/Cli.scala

```diff
@@ -77,10 +77,14 @@ object Cli {
 
     arg[(String, Int)]("[endpoints...]")(endpointRead)
       .action((address, config) => config.copy(participants = config.participants :+ address))
-      .unbounded()
```

Comment:
What is the outcome of this change?

Reply:
:-O Removed the wrong one, thanks for catching!